### PR TITLE
ACN_CD_JuniperSRX_Parser01

### PR DESCRIPTION
**Pull request by chicduong**
## Proposed Changes

  - Juniper SRX Parser Function

### CODE REVIEW

File: Parsers/JuniperSRX/JuniperSRX.txt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+// Using functions in Azure monitor log queries: https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// 
+// LOG SAMPLES:
+// This parser assumes the raw log are formatted as follows:
+//
+// <14>Nov 14 06:13:55 JuniperSRX_FW4 RT_FLOW session created 172.20.22.237/161->134.207.87.33/162 0x0 None 155.117.69.171/52344->134.207.87.33/162 0x0 source rule source-nat-rule N/A N/A 17 trust_to_Untrust_Internet trust untrust 25620 N/A(N/A) reth4.0 UNKNOWN UNKNOWN UNKNOWN
+// <37>Nov 14 06:14:33 JuniperSRX_FW1 sshd Login failed for user 'root' from host '123.45.67.89'
+// <38>Nov 14 06:13:47 JuniperSRX_FW2 sshd[42227] Disconnected from 183.13.25.46 [preauth]
+// 
+Syslog
```
Comment (shainw) [inline Parsers/JuniperSRX/JuniperSRX.txt:28]: You will need a let statement here.  It looks like you want -

let LogHeader = Syslog ...

File: Parsers/JuniperSRX/JuniperSRX.txt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+//
+// REFERENCES: 
+// Using functions in Azure monitor log queries: https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// 
+// LOG SAMPLES:
+// This parser assumes the raw log are formatted as follows:
+//
+// <14>Nov 14 06:13:55 JuniperSRX_FW4 RT_FLOW session created 172.20.22.237/161->134.207.87.33/162 0x0 None 155.117.69.171/52344->134.207.87.33/162 0x0 source rule source-nat-rule N/A N/A 17 trust_to_Untrust_Internet trust untrust 25620 N/A(N/A) reth4.0 UNKNOWN UNKNOWN UNKNOWN
+// <37>Nov 14 06:14:33 JuniperSRX_FW1 sshd Login failed for user 'root' from host '123.45.67.89'
+// <38>Nov 14 06:13:47 JuniperSRX_FW2 sshd[42227] Disconnected from 183.13.25.46 [preauth]
```
Comment (shainw) [inline Parsers/JuniperSRX/JuniperSRX.txt:26]: It would be good to have a sample for RT_IDS.

**Review by shainw — CHANGES_REQUESTED** [review]

File: Parsers/JuniperSRX/JuniperSRX.txt (hunk 1 above)
Comment (chicduong) [reply Parsers/JuniperSRX/JuniperSRX.txt]: updated and submitted

File: Parsers/JuniperSRX/JuniperSRX.txt (hunk 2 above)
Comment (chicduong) [reply Parsers/JuniperSRX/JuniperSRX.txt:26]: Sample added.

**Review by shainw — APPROVED** [review]

**Pull request merged**